users/README.md

```diff
@@ -101,6 +106,11 @@
     "restricted": false,
     "super_user": false,
     "in_discord" : true,
+  },
+    "archivedDetails": {
+    "updated_at": "July 10, 2023 at 8:51:30 AM UTC+",
```

Comment:
Why is it not a unix timestamp because we are saving time in unix timestamp everywhere?

Reply:
updated to UNIX based timestamp